### PR TITLE
Update dependencies to remove vulnerabilities

### PR DESCRIPTION
**Pull request by miketalley**
## Description and Context
Updating dependencies to remove vulnerabilities as referenced in #593 
- Root repo
  - [eslint](https://github.com/eslint/eslint) `6.5.1` -> `7.32.0` ([7.0.0 release notes](https://github.com/eslint/eslint/releases/tag/v7.0.0))
    - I did not upgrade to `8.3.0`([8.0.0 release notes](https://github.com/eslint/eslint/releases/tag/v8.0.0)) because it requires node >= 12, and we currently only require node >= 10, so this requires more thought
- @hubspot/cli
  - [inquirer](https://github.com/SBoudrias/Inquirer.js/) `6.3.1` -> `8.2.0` ([7.0.0 release notes](https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%407.0.0), [8.0.0 release notes](https://github.com/SBoudrias/Inquirer.js/releases/tag/inquirer%408.0.0))
  - [update-notifier](https://github.com/yeoman/update-notifier) `3.0.1` -> `5.1.0` ([4.0.0 release notes](https://github.com/yeoman/update-notifier/releases/tag/v4.0.0), [5.0.0 release notes](https://github.com/yeoman/update-notifier/releases/tag/v5.0.0))
  - [chalk](https://github.com/chalk/chalk) `4.1.0` -> `4.1.2`
    - I did not upgrade to `5.0.0`([release notes](https://github.com/chalk/chalk/releases/tag/v5.0.0)) because it requires node >= 12, and we currently only require node >= 10, so this requires more thought

[spinnies](https://github.com/jcarpanelli/spinnies) contains a vulnerability that was added since `0.3.2` (current version is `0.5.1`). Since we rely on some functionality that has been added, we will leave this version for now and see if we can get a PR to fix the issue in the spinnies repo.

[Lerna](https://github.com/lerna/lerna) contains most of the remaining vulnerabilities, but has not been updated since `4.0.0`, which we are using. There is an [issue to update the dependencies to remove vulnerabilities](https://github.com/lerna/lerna/pull/2986), which is promising.

## Screenshots
Output of `yarn audit` before this PR
image

Output of `yarn audit` after this PR
image

## TODO
- Create a PR for spinnies to fix the vulnerability in their deps @anthmatic 
- Keep an eye on new releases of lerna that include updated dependencies
- Discuss upgrade path to node >= 12

## Who to Notify
@arinker @gcorne @brandenrodgers @kemmerle

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -11,19 +11,19 @@
...
     "findup-sync": "^4.0.0",
     "fs-extra": "^8.1.0",
-    "inquirer": "^6.3.1",
+    "inquirer": "^8.2.0",
     "moment": "^2.29.1",
     "open": "^7.0.3",
     "ora": "^4.0.3",
     "shelljs": "0.8.3",
-    "spinnies": "^0.5.1",
+    "spinnies": "^0.3.2",
```
Comment (anthmatic) [inline packages/cli/package.json:23]: In testing, I think this might cause an issue. I'll see if I can get to the bottom of it

Comment (miketalley) [reply packages/cli/package.json:23]: What was the issue? I tested it out a bit and it seemed to work for me.

Comment (anthmatic) [reply packages/cli/package.json:23]: Getting `TypeError: spinnies.remove is not a function` when running `hs project upload`. Looks like the remove method was added in a later version

**Comment by brandenrodgers** [comment]
This all lgtm aside from the issue with `spinnies.remove`

**Pull request merged**